### ISSUE TITLE
Add JMX arguments to version.json in modpack.jar

### ISSUE THREAD

**Issue by notcake** [notcake/glua-minecraft-tools#4]

**Comment by notcake** [comment]
```
-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.local.only=true -Dcom.sun.management.jmxremote.port=9010
```

**Comment by notcake** [comment]
and remove the `META-INF` directory or else the jar won't validate.

**Comment by notcake** [comment]
Implemented in 62f71c48536a4eca009428d628bd1b656afa10d8
